hack/make/.build-deb/control

```diff
@@ -6,7 +6,7 @@ Vcs-Git: git://github.com/docker/docker.git
 
 Package: docker-engine
 Architecture: linux-any
-Depends: iptables, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Depends: iptables, debhelper, dh-apparmor, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
```

Comment:
Right, our build-depends lives in the Dockerfiles directly, not in the
package metadata. :+1: